.github/workflows/tests.yaml

```diff
@@ -25,21 +25,34 @@ jobs:
           cd api
           DJANGO_SECRET_KEY="testkey" python3 manage.py test
 
+  build-snap:
+    name: Build snap
+    uses: canonical/data-platform-workflows/.github/workflows/build_snap.yaml@v16
```

Comment:
Is it fine to use the data-platform workflow for building snaps ?
In the sense that, do they support it outside of their team use ?

Reply:
Hmm, it's true that only they are using it. I thought that it would be a good idea to avoid reinventing the wheel.  
We'll know if it stops working no ?

Reply:
I asked Dragomir from the data-platform team and he said that there is no issue using it: they will continue to maintain it.